crates/relayer/src/link/cli.rs

```diff
@@ -110,10 +110,16 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Link<ChainA, ChainB> {
             sequences.retain(|seq| sequence_filter.iter().any(|range| range.contains(seq)));
         }
 
+        // Retain only sequences which should not be filtered out
```

Comment:
Don't we also need to filter in `relay_ack_packet_messages_with_packet_data_query_height` below?